prediction_market_agent/agents/microchain_agent/market_functions.py

```diff
@@ -311,16 +295,25 @@ def __call__(self) -> list[str]:
         return [str(position) for position in positions]
 
 
-# Functions that interact with the prediction markets
-MARKET_FUNCTIONS: list[type[MarketFunction]] = [
-    GetMarkets,
-    GetMarketProbability,
-    # PredictProbabilityForQuestionRemote, # Quite slow, use local version for now
-    PredictProbabilityForQuestionLocal,
-    GetBalance,
-    BuyYes,
-    BuyNo,
-    SellYes,
-    SellNo,
-    GetLiquidPositions,
-]
+def build_market_functions(market_type: MarketType) -> list[MarketFunction]:
+    # Functions that interact with the prediction markets
+    market_functions: list[type[MarketFunction]] = [
+        GetMarketProbability,
```

Comment:
The rest will all get moved to `prediction_market_agent.tools.prediction_market` too - this is just a to show an example of what moving one looks like